### PR TITLE
n-queen in python3 using backtracking algorithm.

### PR DESCRIPTION
**Pull request by giTan7**
n-queen in python3 using backtracking algorithm.

### CODE REVIEW

File: other/n-Queen.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Python3 program to solve N Queen  
+# Problem using backtracking 
+global N 
+N = 4
+  
+def printSolution(board): 
+    for i in range(N): 
+        for j in range(N): 
+            print (board[i][j], end = " ")
```
Comment (cclauss) [inline other/n-Queen.py]: Please remove the space between __print__ and __(__.  It is a function just like __range()__.

File: other/n-Queen.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+# solutions, this function prints one of the 
+# feasible solutions. 
+def solveNQ(): 
+    board = [ [0, 0, 0, 0], 
+              [0, 0, 0, 0], 
+              [0, 0, 0, 0], 
+              [0, 0, 0, 0] ] 
+  
+    if solveNQUtil(board, 0) == False: 
+        print ("Solution does not exist")
```
Comment (cclauss) [inline other/n-Queen.py]: Remove space.

**Comment by cclauss** [comment]
Automated tests are failing!

**Comment by giTan7** [comment]
Okay, I will make the changes soon :)

**Comment by cclauss** [comment]
https://travis-ci.org/TheAlgorithms/Python/builds/595024549#L346

File: other/N_QUEEN.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
+"""The n queens puzzle"""
+  class NQueens:
```
Comment (cclauss) [inline other/N_QUEEN.py]: This line must have no leading spaces.

**Comment by cclauss** [comment]
Please format the code with https://github.com/psf/black as discussed in CONTRIBUTING.md or manually ensure that lines 4, 10, 16, 34, 47, and 61 all have four spaces of indentation.

**Comment by cclauss** [comment]
Please get this code to run on your local computer.

Once it is working locally then copy and paste it into this PR.

**Pull request closed without merging**